types/index.d.ts

```diff
@@ -94,7 +94,7 @@ export type Actions = DynamicActionsFunction | ActionType[];
 
 export type CustomActionFunction = (
 	answers: object,
-	config?: ActionConfig,
+	config?: AddActionConfig,
```

Comment:
I'm not convinced that this is the way to go. Is there any reason we can't move `path` into `ActionConfig` instead?

Reply:
According documentation only on add the `path` is used
https://plopjs.com/documentation/#add